src/HttpServer.h

```diff
@@ -5,48 +5,71 @@ All rights reserved.
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file. See the AUTHORS file for names of contributors.
  */
+#include <boost/array.hpp>
+#include <boost/asio.hpp>
+#include <boost/bind/bind.hpp>
+#include <boost/enable_shared_from_this.hpp>
+#include <boost/shared_ptr.hpp>
+#include <ctime>
 #include <iostream>
+#include <string>
 
 #include "ThreadPool.h"
 
 #ifndef SHORTLINK_SRC_HTTPSERVER_H_
 #define SHORTLINK_SRC_HTTPSERVER_H_
 
 namespace shortlink {
-
+enum ServerStatus { kRunning, kStopped };
```

Comment:
Please add a comment on what this is for.

Reply:
comment added.